src/AppHeader/AppHeader.test.js

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from "@testing-library/react";
+
+import AppHeader from "./AppHeader";
+import React from "react";
+
+// a set of default inputs so that tests can change what theyre testing
+const defaultInputs = {
+  onChangePassword: jest.fn(),
+  onLogout: jest.fn(),
+  onModeChange: jest.fn(),
+  username: "Joe Bloggs"
+};
+
+// test app name in the component
+describe("AppHeader", () => {
+  test("should render app name", () => {
```

Comment:
The tests for the app header component are somewhat lacking in coverage...

We can do better here. Please create a separate task to improve this after the code freeze.